target_chains/solana/pyth_solana_receiver_sdk/src/cpi/accounts.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::vec_init_then_push)]
```

Comment:
couldn't we move this in?

Reply:
The module contains auto generated code so I feel like it's more convenient to keep this outside so it's easier to update the code if needed.